### DISCUSSION TITLE
edge: dot "." based path seperators are depricated, We recommend using "/" instead Deprecation\Warning

### DISCUSSION THREAD

**Discussion by theashywolf** [adonisjs/core#2021] [Help] [answered]
![image](https://user-images.githubusercontent.com/31142505/101454162-dc997980-3973-11eb-8876-1c90da8de822.png)

I need help over here!! it appears many exceptions and goes crash
its something was happend but i have no idea to fix!
WHat should id do?!

**Comment by WeyIin** [comment, marked as answer]
I had the same problem and fixed it by changing `.` to `/`. Literally what the console says.

Example in your controller:
```
return view.render('home.index')
```

change to:
```
return view.render('home/index')
```

Literally change all occurrences in all controllers where you serve your view.

Hope that helps 👍

**Reply by theashywolf** [reply]
You just saved my day! 👍

**Reply by WeyIin** [reply]
Glad you managed to sort it out :)
